altair/vegalite/v2/examples/interval_selection.py

```diff
@@ -0,0 +1,28 @@
+"""
+Interval Selection Example
+==========================
+
+This is an example of creating a stacked chart for which the domain of the
+top chart can be selected by interacting with the bottom chart.
+"""
+
+import altair as alt
+from vega_datasets import data
+sp500 = data.sp500()
+
+brush = alt.selection(type='interval', encodings=['x'])
+
+upper = alt.Chart().mark_area().encode(
+    x=alt.X('date', scale={'domain': brush.ref()}),
+    y='price'
+).properties(
+    width=600,
+    height=200
+)
+
+lower = upper.properties(
+    selection=brush,
+    height=60
+)
+
+chart = alt.VConcatChart([upper, lower], data=sp500)
```

Comment:
`VConcatChart` is a bit verbose -- I guess you would have `VConcat` or even `V` (that @ellisonbg proposed) as aliases?

Reply:
Well, in the schema it's ``TopLeve<lVConcatSpec>``, so I thought ``VConcatChart`` was a nice concise alias 😀

Reply:
And yes, ``&`` works as well, but it's less convenient to specify data if you use ``&``

Reply:
I think in the end I'd probably keep the object name something like ``VConcatChart``, but provide an ``alt.vconcat`` function to make things more concise.

Reply:
This sounds good.  Also, thanks for bringing up about `TopLevel<VConcatSpec>`. 

Now I'm also curious how Altair deals with the fact that [some properties such as `autosize` and `config`   are only supported at the `TopLevel`](https://vega.github.io/vega-lite/docs/spec.html#top-level-specifications).

(That's why we have `VConcatSpec` and `TopLevel<VConcatSpec>` -- the latter is for the top-level spec only -- when nested, only the former can be used.)

Reply:
Yeah, I haven't really figured that out yet. Any ideas? :smile:

Reply:
I added ``vconcat()``, ``hconcat()``, and ``layer()`` functions in https://github.com/altair-viz/altair/pull/476